test/utils/download.ts

```diff
@@ -0,0 +1,92 @@
+import { expect } from 'chai';
+import fs from 'fs';
+import * as axios from 'axios';
+import * as downloadUtil from '../../src/utils/download';
+import * as workerProcess from '../../src/utils/worker-process';
+
+describe('download utils', () => {
+	let execStub: any = null;
+	beforeEach(() => {
+		execStub = sandbox.stub(workerProcess, 'exec');
+	});
+
+	describe('#download', () => {
+		let existsSyncStub: any = null;
+
+		beforeEach(() => {
+			sandbox.stub(axios);
+			existsSyncStub = sandbox.stub(fs, 'existsSync');
+			sandbox.stub(fs, 'createWriteStream').returns(Buffer.alloc(1));
+		});
+
+		it('should return if download already exists', () => {
+			existsSyncStub.returns(true);
+
+			return expect(downloadUtil.download('url', 'file/path')).returned;
+		});
+
+		it.skip('should get latest version', async () => {
```

Comment:
why skip?

Reply:
`axios` default stubbing not working, so I need help to fix, lets sit together when u have time?